tests/test_cachedataset_persistent_workers.py

```diff
@@ -19,15 +19,14 @@
 @SkipIfBeforePyTorchVersion((1, 7))
 class TestTransformsWCacheDatasetAndPersistentWorkers(unittest.TestCase):
     def test_duplicate_transforms(self):
-        im, _ = create_test_image_2d(128, 128, num_seg_classes=1, channel_dim=0)
-        data = [{"img": im} for _ in range(2)]
+        data = [{"img": create_test_image_2d(128, 128, num_seg_classes=1, channel_dim=0)[0]} for _ in range(2)]
 
         # at least 1 deterministic followed by at least 1 random
         transform = Compose([Spacingd("img", pixdim=(1, 1)), RandAffined("img", prob=1.0)])
 
         # cachedataset and data loader w persistent_workers
         train_ds = CacheDataset(data, transform, cache_num=1)
-        train_loader = DataLoader(train_ds, num_workers=2, persistent_workers=True)
+        train_loader = DataLoader(train_ds, num_workers=1, persistent_workers=True)
```

Comment:
May I know why changing the test workers from 2 to 1?
What't the root cause of test issue?

Thanks.

Reply:
Hi @wyli ,

Have you found how to reproduce it locally?

Thanks.

Reply:
No I haven't

Reply:
thanks, I added inline comments for future investigations.